.github/workflows/docker.yml

```diff
@@ -64,8 +50,8 @@ jobs:
           ./scripts/docker_build_push.sh "" ${{ matrix.target }} ${{ matrix.platform }}
 
   ephemeral-docker-build:
-    needs: config
-    if: needs.config.outputs.has-secrets
+    needs: check-secrets
+    if: needs.check-secrets.outputs.has-secrets == 'true'
```

Comment:
Not totally related with this PR, but I think it only makes sense to build and push to a local file on the workspace workflow while the PR is open, so this means we are supposed to not have secrets here. the `ephemeral-env` workflow is expecting the build dir